src/pages/explore/masternodes/SvgIconsColumn.tsx

```diff
@@ -121,7 +139,10 @@ function FeatureIcon({
       <div
         className={classNames(
           "space-y-4 md:ml-0 md:mt-5",
-          customParagraphStyle
+          customParagraphStyle,
+          {
+            "md:mx-auto md:ml-auto lg:text-left lg:mx-0": isCenterAligned,
```

Comment:
i think this is slightly confusing because of `text-left`, can we put this into `customParagraphStyle`?